CSS/media.css

```diff
@@ -0,0 +1,309 @@
+    z-index: 5;
+  }
+
+  .navbar1 {
+    padding-left: 134px;
+  }
+
+  .navbar2 {
+    padding-left: 0;
+  }
+
+  .nv2-img {
+    display: inline-block;
+    width: 114px;
+    height: 48px;
+    background-image: url("../Images/tnw_navbar.png");
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  .nav2-ul {
+    margin-left: 20px;
+  }
+
+  .nv2-div-tnw,
+  .nv2-div-clock,
+  .nv2-bttn {
+    display: none;
+  }
+
+  .navbar3 {
+    padding-left: 134px;
+  }
+
+  .grid-1 {
+    height: 366px;
+  }
+
+  .grd1-big {
+    background:
+      linear-gradient(170deg, transparent, rgba(0, 0, 0, 0.8)),
+      url(../Images/grid1_big_image.jpg);
+    padding: 14px;
+    position: relative;
+  }
+
+  .big-logo-date {
+    display: none;
+  }
+
+  .big-title {
+    font-size: 40px;
+    line-height: 43px;
+  }
+
+  .grd1-author {
+    position: absolute;
+    bottom: 14px;
+  }
+
+  .grd1-side-title {
+    font-size: 18px;
+    line-height: 21px;
+  }
+
+  .to-bottom {
+    bottom: 14px;
+  }
+
+  .to-bottom .grd1-author {
+    position: unset;
+  }
+
+  .chair {
+    padding: 8px;
+    border-radius: unset;
+    font-size: 16px;
+    color: rgb(255, 255, 255);
+    position: fixed;
+    right: 0;
+    bottom: 0;
+    z-index: 10;
+  }
+
+  .chair:hover {
+    font-size: 20px;
+  }
+}
+
+@media screen and (max-width: 990px) {
+  .grid-titles-cont {
+    flex-direction: column;
+    justify-content: unset;
+    align-items: unset;
+  }
+
+  .funding-info {
+    margin-top: 10px;
+  }
+
+  .title-option {
+    margin-top: 10px;
+  }
+
+  .grid-2 {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(3, auto);
+    column-gap: 20px;
+    row-gap: 23px;
+  }
+
+  .grid-3 {
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr;
+  }
+
+  .grd3-item4 {
+    display: none;
+  }
+
+  .grd3-company {
+    height: 43px;
+  }
+
+  .grid-4 {
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: repeat(5, auto);
+    column-gap: 20px;
+  }
+
+  .grd4-latest-img {
+    width: 96px;
+    padding-top: 96px;
+  }
+}
+
+@media screen and (max-width: 856px) {
+  .menu-bars {
+    display: flex;
+    height: 100%;
+    justify-content: center;
+    align-items: center;
+    border-left: 1px solid rgb(235, 235, 235);
+  }
+
+  .fa-bars {
+    font-size: 16px;
+    color: orange;
+    padding: 0 16px;
+  }
+
+  .nav2-ul {
+    display: none;
+  }
+}
+
+@media screen and (max-width: 790px) {
+  .corner-tnw-img {
+    display: none;
+  }
+
+  .navbar1 {
+    display: none;
+  }
+
+  .navbar2 {
+    position: fixed;
+    top: 0;
+  }
+
+  .navbar3 {
+    height: 60px;
+    justify-content: center;
+    align-items: center;
+    padding: 0;
+    position: fixed;
+    top: 48px;
+  }
+
+  .grid-1 {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: repeat(3, 1fr);
+    column-gap: 8px;
+    row-gap: 8px;
+    height: auto;
+    margin: 110px 7px 50px;
+  }
```

Comment:
**[OPTIONAL]** On smaller screens the first article within grid-1 seems to have issues. Kindly fix it.

Hint: Play around with max-height on `.grid-1` to check which works best.